Automattic/simplenote-ios
# Drops 1Password Framework

### Details
Since our deployment target (12!) supports Password AutoFill, it's time to drop the 1Password Framewor 😃

@etoledom Eduardooo!! Would you be game for a beautiful PR?

Thank youuuu!!!


### Test: Login
0. Make sure Password Autofill is Enabled: `Settings.app > Passwords > Autofill > 1Password`
1. Fresh install the app
2. Press over **Login  > Log In with Email**

- [x] Verify that Password Autofill shows up, immediately, above the keyboard
- [x] Verify that pressing the key icon lets you open 1Password

### Test: Login
0. Make sure Password Autofill is Enabled: `Settings.app > Passwords > Autofill > 1Password`
1. Fresh install the app
2. Press over **Sign Up**

- [x] Verify that Password Autofill shows up, immediately, above the keyboard
- [x] Verify that pressing the key icon lets you open 1Password

### Release
`RELEASE-NOTES.txt` was updated in 3e39083 with:
 
> Password AutoFill is now the preferred way to manage your credentials
